### PR TITLE
update branding

### PR DESCRIPTION
**Pull request by ttlew**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Environment variables
 WORKSPACE_ID=<workspace-id>
-CONVERSATION_USERNAME=<username>
-CONVERSATION_PASSWORD=<password>
+WATSON_ASSISTANT_USERNAME=<username>
```
Comment (germanattanasio) [inline .env.example]: `ASSISTANT_USERNAME` not `WATSON_ASSISTANT_USERNAME`

File: README.md [hunk 2]
```diff
@@ -143,11 +143,11 @@ If you want to experiment with the application or use it as a basis for building
    }
    ```
 
-1. In the JSON output, find the values for the `password` and `username` keys. Paste these values (not including the quotation marks) into the `CONVERSATION_PASSWORD` and `CONVERSATION_USERNAME` variables in the `.env` file:
+1. In the JSON output, find the values for the `password` and `username` keys. Paste these values (not including the quotation marks) into the `WATSON_ASSISTANT_PASSWORD` and `WATSON_ASSISTANT_USERNAME` variables in the `.env` file:
```
Comment (germanattanasio) [inline README.md:147]: `ASSISTANT_USERNAME`

File: manifest.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 declared-services:
-  conversation-prod:
```
Comment (germanattanasio) [inline manifest.yml:3]: This should match the README

```
     - watson-assistant-ce-tone
     - tone-analyzer-ce-tone
```

**Review by germanattanasio — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ttlew) [reply README.md:147]: updated

File: manifest.yml (hunk 3 above)
Comment (ttlew) [reply manifest.yml:3]: updated

**Review by germanattanasio — APPROVED** [review]

**Pull request merged**